doc/classes/CanvasItem.xml

```diff
@@ -584,14 +585,16 @@
 			<return type="void" />
 			<param index="0" name="enable" type="bool" />
 			<description>
-				If [param enable] is [code]true[/code], this node will receive [constant NOTIFICATION_LOCAL_TRANSFORM_CHANGED] when its local transform changes.
+				If [code]true[/code], the node will receive [constant NOTIFICATION_LOCAL_TRANSFORM_CHANGED] whenever its local transform changes.
+				[b]Note:[/b] Many canvas items such as [Camera2D] or [CollisionShape2D] automatically enable this in order to function correctly.
```

Comment:
You could give different example nodes for local and global transform (unless they use both notifications?).

Reply:
Intriguing.
**Camera2D** needs `set_notify_transform` (global) and **CollisionShape2D needs  `set_notify_local_transform`. Broadly speaking, one could "lie" to keep the note consistent across both, but... it would be better to figure out two more nodes to provide as examples.

Reply:
There's very few nodes that are initialized with `set_notify_transform` or `set_notify_local_transform`. Many call these method later, in ways that may need to be documented individually. Not sure if making that fact clearer, but I have updated the examples.